spec/models/collection_spec.rb

```diff
@@ -44,7 +46,17 @@
         allow(@collection).to receive(:members).and_return([file, file, file])
         allow(ActiveFedora::SolrService).to receive(:query).with(query, args).and_return(documents)
       end
-      it { is_expected.to eq 99 }
+
+      context "saved" do
+        before do
+          allow(@collection).to receive(:new_record?).and_return(false)
+        end
+        it { is_expected.to eq 99 }
+      end
+
+      context "not saved" do
+        it { expect {subject}.to raise_error RuntimeError }
```

Comment:
Your first line does not work.  The error gets raise on the subject line and never caught.  That was the first syntax I tried, and then banged my head trying to get one that works.